src/components/UserManagers.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  userManagerNames: string
+}
+
+const UserManagers = ({ userManagerNames }: Props) => {
+  return (
+    <p className="govuk-body">
+      {`If you have any queries about your permissions or cannot see the resources you expect, please contact one of the user managers for your force: ${userManagerNames}.`}
+    </p>
+  )
+}
```

Comment:
FWIW this can be slightly simplified (omit the return and the braces) to:

const UserManagers = ({ userManagerNames }: Props) => (
    <p className="govuk-body">
      {`If you have any queries about your permissions or cannot see the resources you expect, please contact one of the user managers for your force: ${userManagerNames}.`}
    </p>
  )

Reply:
yup, good point. I initially wanted to do more things here, hence the return still being here.